Screens/SettingsScreenComponents/ChooseLanguage.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { Text, StyleSheet, View, Picker, TouchableOpacity, Alert } from 'react-native';
+import DeviceInfo from "react-native-device-info";
+import { connect } from 'react-redux';
+import { storeSettings } from '../../Actions/settingsActions';
+import LocalizeComponent from '../../Localization/LocalizedComponent';
+
+class ChooseLanguage extends LocalizeComponent {
+    state = {
+        settings: { language: "" }
+    }
+
+    componentDidUpdate(prevState) {
+        if (!prevState.settings.error && this.props.settings.error) {
+            Alert.alert(this.t('Error!'), this.t('Please try again'));
```

Comment:
don't use alert
Add control on screen that will get error from store

Reply:
Good idea